#ifndef ADDDEVICE_DIALOG_H
#define ADDDEVICE_DIALOG_H
#include <mainwindow.h>
#include <QDialog>

namespace Ui {
class AddDevice_dialog;
}

class AddDevice_dialog : public QDialog
{
    Q_OBJECT

public:
    explicit AddDevice_dialog(QWidget *parent = nullptr);
    ~AddDevice_dialog();

private slots:
    void on_add_device_button_clicked();

    void on_cancel_button_clicked();

    void on_AddDevice_dialog_finished(int result);

private:
    Ui::AddDevice_dialog *ui;
    MainWindow *mw;
};

#endif // ADDDEVICE_DIALOG_H
